acmwebsite/lib/helpers.py

```diff
@@ -12,40 +12,52 @@
 from acmwebsite.lib.mailmanapi import ListAdminAPI
 mmadmin = ListAdminAPI(tg.config.get('mailman.url'), tg.config.get('mailman.secret'))
 
+
 def current_year():
     now = datetime.now()
     return now.strftime('%Y')
 
+
 def markdown(*args, strip_par=False, **kwargs):
     res = md.markdown(*args, **kwargs)
     if strip_par:
         res = res.replace('<p>', '').replace('</p>', '')
     return Markup(res)
 
+
 def icon(icon_name):
     return Markup('<i class="glyphicon glyphicon-%s"></i>' % icon_name)
 
-def ftime(datetime_obj, show_day=False):
+
+def ftime(datetime_obj, duration=None, show_day=False):
     day_fmt = '{0:%A}, ' if show_day else ''
     date_fmt = '{0.day} {0:%B %Y}'
     time_fmt = '{0:%H}:{0:%M}'
     if isinstance(datetime_obj, datetime):
-        return (day_fmt + date_fmt + ' at ' + time_fmt).format(datetime_obj)
+        if not isinstance(duration, timedelta):
```

Comment:
A brief comment explaining what this block of code does would be good. For example, something that gives an example of what string will be outputted when there is a duration vs. when there isn't.